Dockerfile

```diff
@@ -4,23 +4,16 @@ FROM rust:1.73-bookworm AS builder
 WORKDIR /usr/src/glaredb
 COPY . .
 
-RUN apt-get update && apt-get install -y openssl ca-certificates
```

Comment:
(review): I'm expecting that we only needed these to run the openssl x509 gen. If a build dependency relies on `ca-certificates`, I'd want to know why (and to ensure it's documented).

Reply:
If it builds without it, then probably fine. I could see trying to fetch the protoc binary potentially fail without ca certs.

Reply:
CI build step passed, and I succesfully `docker build .` locally.